dc_make/maker.py

```diff
@@ -116,8 +128,54 @@ def getInputFile(self, era, model_params):
       self.input_files[file_name] = file
     return file_name, self.input_files[file_name]
 
+  def getRelevantBins(self, era, channel, category,signal_processes_histograms,unc_name=None, unc_scale=None, model_params=None):
+    relevant_bins = []
+    for hist in signal_processes_histograms:
+        axis = hist.GetXaxis()
+        hist_integral = hist.Integral(1,axis.GetNbins() + 1)
+        for nbin in range(1,axis.GetNbins() + 1):
+          isImportant=False
+          if hist_integral !=0 and hist.GetBinContent(nbin) / hist_integral >= 0.2:
+            isImportant = True
+          if len(relevant_bins) < axis.GetNbins():
+            relevant_bins.append(isImportant)
```

Comment:
I don't understand how it works with the list of booleans.

Reply:
at the beginning there was a different definition of resolve negative bins that needed list of booleans. I fixed it to make it work for `resolveNegativeBins`